sdk/storage/storage-queue/ChangeLog.md

```diff
@@ -1,6 +1,21 @@
 # Changelog
 
-## 2019.11 12.0.0-preview.5
+## 2019.11 12.0.0-preview.6
+
+- [Breaking] The interface `NewPipelineOptions` has been renamed to `StoragePipelineOptions` and its
+properties have been updated as below:
+    - The `proxy` property of type `ProxySettings | string` has been renamed to `proxyOptions` and
+    will be of type `ProxyOptions`. The helper function `getDefaultProxySettings()` can be used to 
```

Comment:
>getDefaultProxySettings [](start = 57, length = 23)

Where does it be exported?

Reply:
It is exported from `core-http` library.

Reply:
Maybe we should remove the note on `getDefaultProxySettings()` here.
The main intention is to encourage users to pass the `ProxyOptions`

@bterlson thoughts?

Reply:
I'm ok removing the note. It's probably easier to split the url into host/port than to find getDefaultProxySettings and use it.

Reply:
Ok. Please remove for file/blob too. 

Actually, when some APIs exported in core-http. If we encourage storage customer using it, we'd better export it from storage SDK. Otherwise, customer needs to depend on core-http directly too.

Reply:
Pushed a commit to replace the note on using `getDefaultProxySettings` with a note on splitting the url and passing `host` and `port`